juju/juju
# test: fix charms without manifest

In 4.0, all charms now require a manifest. Many of our tests were using the `jameinel-ubuntu-lite` charm, which doesn't not have a manifest. In most cases, we can simply replace this with `ubuntu-lite` without changing the test. However, a few tests involve refreshing a charm, and so we need a charm with multiple revisions - I chose `juju-qa-test`.

## QA steps

Run affected test suites. They may fail for other reasons, but at least we should not see an error such as the following:
```
charm without manifest not valid
```

## Links

**Jira card:** JUJU-6880
